app/_kong_plugins/bot-detection/index.md

```diff
@@ -32,6 +32,25 @@ categories:
 
 search_aliases:
   - bot-detection
+
+
 ---
 
-## Overview
+Using the Bot Detection plugin, you can protect your a Gateway Service or a Route from bots. It automatically detects [common bots](https://github.com/Kong/kong/blob/master/kong/plugins/bot-detection/rules.lua) on every request from the associated Gateway Service or Route using regex.
+
+You can also configure custom regex patterns to either allow or deny additional bots. 
+
+## How it works
+
+Once the Bot Detection plugin is enabled on a Gateway Service or Route, it checks the `User-Agent` header of incoming requests. If the header matches a default common bot or any custom denied bot regex you've configured, the request is immediately blocked with a `403` response.
+
+If the `User-Agent` header doesn't match any common or denied bots, the request is processed. This also applies when a configured allowed bot matches.
+
+## Use case
+
+If you suspect bot traffic you can use a logging plugin like [File Log](/plugins/file-log/) on a Gateway Service to track down unusual `User-Agent` headers in incoming requests. 
```

Comment:
```suggestion
If you suspect bot traffic you can use a [logging plugin](/plugins/?category=logging) on a Gateway Service to track down unusual `User-Agent` headers in incoming requests. 
```
Removing File Log in favor of a link to the category instead, since we have no real reason to recommend a specific logging plugin.